internal/eventstore/handler/handler_projection.go

```diff
@@ -165,7 +165,9 @@ func (h *ProjectionHandler) processEvent(
 	h.lockMu.Lock()
 	defer h.lockMu.Unlock()
 
-	h.stmts = append(h.stmts, stmts...)
+	if stmt != nil {
```

Comment:
IMO should `stmt` only be nil if an `err` occurred

Reply:
changed together with changing to []*Statement